agent-any/src/AnyAgentComplex.cc

```diff
@@ -432,7 +432,20 @@ AnyAgent::parseData (char const *&line, const YCPValue & syntax, bool optional)
     if ((line == 0) || (*line == 0))
 	line = getLine ();
     if (line == 0)
+    {
+        // handle special terms that do not need data like optional data
```

Comment:
Czech "jako" is either "like" or "as".
Here it is parsed as "Handle X" where X includes "data like Y"
but you need to use "as" to parse it as "Handle X as Y".

// If the syntax being parsed does not need any string, then EOF is in fact OK

Please document the API of parseData: return Null if parsing failed, return Void if we parsed nothing successfully (like an Optional syntax)